### DISCUSSION TITLE
Where to put music files?

### DISCUSSION THREAD

**Discussion by junkjunker** [hackthedev/remotepifm#6] [General]
I'm running the App on a Pixel 2, but I don't have any music files listed on the screen.  Where exactly should I be putting the music files?  On the Pixel, or on the Pi?
Thanks!

**Comment by hackthedev** [comment]
On the top right corner should be an upload simbol. there you can select the music file to upload to your pi. It should be `/home/pi/sounds` if i dont make a mistake. Files need to be on the raspberry pi, but the android app lets you upload them from your mobile device
